webapp/src/css/inbox.less

```diff
@@ -27,64 +27,90 @@
 
 body {
   overflow-y: hidden;
-  .app-root.messages {
-    .header .tabs .selected,
-    .filters {
+  .app-root {
+    &.messages .filters {
       background-color: @messages-color;
     }
-  }
-  .app-root.tasks {
-    .header .tabs .selected,
-    .filters {
+    &.tasks .filters {
       background-color: @tasks-color;
     }
-  }
-  .app-root.reports {
-    .header .tabs .selected,
-    .filters {
+    &.reports .filters {
       background-color: @reports-color;
     }
-  }
-  .app-root.analytics {
-    .header .tabs .selected,
-    .filters {
+    &.analytics .filters {
       background-color: @analytics-color;
     }
-  }
-  .app-root.contacts {
-    .header .tabs .selected,
-    .filters {
+    &.contacts .filters {
       background-color: @contacts-color;
     }
-  }
-  .app-root.user {
-    .header .tabs .selected,
-    .filters {
+    &.user .filters {
       background-color: @admin-color;
     }
-  }
-  .app-root.about,
-  .app-root.testing,
-  .app-root.privacy-policy {
-    .header .tabs .selected,
-    .filters {
-      background-color: @help-color;
-    }
-    .filters {
-      height: 0;
-    }
-    .page {
-      top: 75px;
+    &.error .filters,
+    &testing .filters {
+      background-color: @error-color;
     }
-    .partners > h3 {
-      margin-top: 100px;
+
+    &.about,
+    &.testing,
+    &.privacy-policy {
+      .filters {
+        background-color: @help-color;
+        height: 0;
+      }
+      .page {
+        top: 75px;
+      }
+      .partners > h3 {
+        margin-top: 100px;
```

Comment:
What is `top` and `margin-top`? can we have variables for these?

Reply:
<img width="313" alt="Screenshot 2024-08-23 at 15 18 11" src="https://github.com/user-attachments/assets/5a7a515a-fa67-4275-b0bd-0f98ce1d2e6c">
 
 The about page, and privacy-policy dont have the colored filter bar that's why on those pages we adjust the top to move the page up a bit.  perhaps a variable name like this.
 
 ```
 .page {
   top: @top-without-filter;
}
 ```